### DISCUSSION TITLE
Confused with the path

### DISCUSSION THREAD

**Discussion by shreya-gb** [vinitshahdeo/inspirational-quotes#2346] [Q&A] [answered]
Hello, @vinitshahdeo I had a doubt.
While adding a new quote. I'm lost with the git checkout. Please could you tell me where I went wrong?

1. Installed Node.
2. Installed NPM
3. git cloned in local **C://**
4. There was a folder "node_modules/inspirational_quotes"
5. Tried Git check out here, but it throws an error saying (**fatal: not a git repository (or any of the parent directories): .git**)

Is this right?
Or should I have the inspirational quotes in my public user folder instead of within **node_modules**?

**Comment by vinitshahdeo** [comment, marked as answer]
Hey @shreya-gb 

Please clone the repo and proceed.

```
$ git clone https://github.com/vinitshahdeo/inspirational-quotes.git
$ cd inspirational-quotes
$ npm install
```
